dali/operators/image/remap/cvcuda/warp_perspective.cc

```diff
@@ -108,16 +108,16 @@ class WarpPerspective : public nvcvop::NVCVSequenceOperator<StatelessOperator> {
   float4 GetFillValue(int channels) const {
     if (fill_value_arg_.size() > 1) {
       if (channels > 0) {
-        if (channels == static_cast<int>(fill_value_arg_.size())) {
-          float4 fill_value{0, 0, 0, 0};
-          memcpy(&fill_value, fill_value_arg_.data(), sizeof(decltype(fill_value)));
```

Comment:
The bug is in this line - fill_value_arg_ is constructed with exact length in line 95, overriding the default constructor from 233, which is never used. This leads to invalid read of size 16, which is up to 12 bytes beyond the vector end.